jyve/exporter.py

```diff
@@ -42,6 +42,8 @@ class JyveExporter(HTMLExporter):
             "lab/api/settings/@jupyterlab/fileeditor-extension:plugin",
             "lab/api/settings/@jupyterlab/notebook-extension:tracker",
             "lab/api/settings/@jupyterlab/shortcuts-extension:plugin",
+            "kernelspecs/python3/logo-64x64.png",
```

Comment:
This stayed in because the `default_kernel` has to be in the list in the api response to `/api/kernelspecs` or te launcher freaks out. On export need to change that response to say one of the jyve kernels (configurable)... or do some more patching.

Reply:
Makes sense, I'm happy to close this pull request if you like?